### DISCUSSION TITLE
why theres no metamask in web3modal

### DISCUSSION THREAD

**Discussion by jeffohbabygames** [WalletConnect/walletconnect-monorepo#2285] [Developer Support] [answered]
i implemented web3modal in react-native expo but when i open the modal there's no metamask in it

**Comment by iamoracle** [comment, marked as answer]
Metamask is yet to support WC V2.

**Reply by jeffohbabygames** [reply]
thanks sir.

**Reply by nissoh** [reply]
having the same issue and baffled a bit

https://lab.web3modal.com/ManagedReact seems to actually support metamask but https://lab.web3modal.com/StandaloneReact doesn't

at the code level they seem almost similar, am i missing something?

**Reply by nissoh** [reply]
found a way to resolve this, not sure why but after importing @web3modal/html instead of the standalone the injected metamask option is now showing!

@iamoracle @jeffohbabygames

**Comment by finessevanes** [comment]
Hey there - when using the /html and /react version you can toggle between v1 and v2 sign support like this:
```
const wagmiConfig = createConfig({
  autoConnect: true,
  connectors: w3mConnectors({ projectId, version: 1, chains }),
  publicClient
})
```

For v1 support, keep `version: 1` (MetaMask support).

If you're using /standalone:
```
const web3Modal = new Web3Modal({
  walletConnectVersion: 2,
  projectId,
  standaloneChains: ['eip155:1']
})
```
You will want to change `walletConnectVersion` and set it to 1 for v1 support like this: `walletConnectVersion: 1`.
